portfolio/src/main/webapp/script.js

```diff
@@ -72,12 +72,16 @@ async function getComments() {
     const text = await response.json();
     for (const currentText of text) {
         const newText = document.createElement('p');
-        newText.innerHTML = currentText;
+        const newImage = document.createElement('img');
```

Comment:
nit: you can use `Image` constructor to create an `<img>`

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image